src/sonic-config-engine/config_samples.py

```diff
@@ -95,7 +123,9 @@ def generate_l2_config(data):
 _sample_generators = {
         't1': generate_t1_sample_config,
         'l2': generate_l2_config,
-        'empty': generate_empty_config
+        'empty': generate_empty_config,
+        'l1': generate_l1_config,
```

Comment:
Can you please add a unit test for this? You can refer as done for https://github.com/Azure/sonic-buildimage/pull/7215

Reply:
Thanks for the comments. Added UT. Please review.

Reply:
@prsunny do you have any more comments or else would be able to approve this PR ?